client/my-sites/site-settings/form-analytics-stores.jsx

```diff
@@ -53,13 +55,57 @@ class FormAnalyticsStores extends Component {
 		return this.renderSettings( settings, disableAll );
 	};
 
+	renderAdvancedSettings = disableAll => {
+		const { translate } = this.props;
+		const settings = [
+			{
+				key: 'enh_ec_tracking',
+				label: translate( 'Enable Enhanced eCommerce' ),
+				explanation: translate(
+					'Before enabling, turn on enhanced eCommerce in your Google Analytics dashboard.'
+				),
+			},
+			{
+				key: 'enh_ec_track_remove_from_cart',
+				label: translate( 'Remove from Cart events' ),
+				requires: 'enh_ec_tracking',
+			},
+			{
+				key: 'enh_ec_track_prod_impression',
+				label: translate( 'Product impressions from listing pages' ),
+				requires: 'enh_ec_tracking',
+			},
+			{
+				key: 'enh_ec_track_prod_click',
+				label: translate( 'Product clicks from listing pages' ),
+				requires: 'enh_ec_tracking',
+			},
+			{
+				key: 'enh_ec_track_prod_detail_view',
+				label: translate( 'Product detail views' ),
+				requires: 'enh_ec_tracking',
+			},
+			{
+				key: 'enh_ec_track_checkout_started',
+				label: translate( 'Checkout process initiated' ),
+				requires: 'enh_ec_tracking',
+			},
+		];
+
+		return this.renderSettings( settings, disableAll );
+	};
+
 	render = () => {
 		const { disabled, translate } = this.props;
 
+		// TODO config flag it
+
 		return (
 			<div>
 				<FormLegend>{ translate( 'Basic store analytics' ) }</FormLegend>
 				{ this.renderBasicSettings( disabled ) }
+				<FormLegend>{ translate( 'Advanced store analytics' ) }</FormLegend>
```

Comment:
Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'Basic store analytics' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).